src/Microsoft.DocAsCode.Plugins/ISupportIncrementalPostProcessor.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Plugins
+{
+    public interface ISupportIncrementalPostProcessor
+    {
+        /// <summary>
+        /// Get post processor host
+        /// </summary>
+        IPostProcessorHost PostProcessorHost { get; set; }
```

Comment:
but if removing setter, outer code couldn't set the host except for the processor itself?

---
In reply to: [94111780](https://github.com/dotnet/docfx/pull/1081#discussion_r94111780) [](ancestors = 94111780)